add article to announce discord channel and website for advent of code (#1576)
scala/scala-lang: blog/_posts/2023-11-23-advent-of-code-announce.md

---
layout: blog-detail
post-type: blog
by: Jamie Thompson, Scala Center
title: "Join us for the Advent of Code 2023"
---

### Summary
We are happy to announce that for the third year in a row we are supporting the Scala Community's participation in the annual [Advent of Code](https://adventofcode.com/) challenge for 2023! _(Look [here](https://scala-lang.org/blog/2023/01/06/advent-of-code-recap.html) for a recap of 2022's edition)._

> Advent of Code, organised by [Eric Wastl](http://was.tl/) is a yearly event consisting of a series of small programming puzzles. The puzzles are released daily from December 1st to December 25th. The goal is to improve your understanding of your programming language of choice by solving as many puzzles as possible.

### Discord Channel

> Since 2021, the Scala Center has begun a tradition of participating in the challenge.

Reviewer: ```suggestion
In 2021, the Scala Center has begun a tradition of participating in the challenge.
```

This year, we invite you to join us again!
We have re-opened the `advent-of-code` [Discord channel](https://discord.com/channels/632150470000902164/913451015246868530) at the official [Scala Discord Server](https://discord.com/invite/scala).
Here you can discuss the puzzles, share solutions, and ask for help.

### Read and share solutions on our website

We have also created a new section for the 2023 edition on the [Scala Center Advent of Code website](https://scalacenter.github.io/scala-advent-of-code/2023/).
Starting December 1st, we will post a daily article corresponding to the day’s puzzle (ending on December 25th).

If you want to submit a puzzle solution to our website, click “Edit this page” on a puzzle page, and then create a pull request with the link to your solution under the "Solutions from the community" section.

> Otherwise, members of the Scala Center and other volunteers, will try to solve the solution ourselves and post the code on each day’s article with an explanation.

Reviewer: ```suggestion
Otherwise, members of the Scala Center and other volunteers will try to solve the solution themselves and post the code in each day’s article with an explanation.
```
